data/core/common.lua

```diff
@@ -1,22 +1,43 @@
 local common = {}
 
 
+---Checks if the byte at offset is a UTF-8 continuation byte.
+---
+---UTF-8 encodes code points in 1 to 4 bytes. </br>
+---The first byte starts with a number of 1 bits followed by 0, </br>
+---where the number of 1 bits represent the number of bytes needed for the sequence. </br>
+---For multi-byte sequences, each byte following the start byte starts with the bits 10.
```

Comment:
I was thinking about something simpler, kinda like
```suggestion
---UTF-8 encodes code points in 1 to 4 bytes. </br>
---For multi-byte sequences, each byte following the start byte is a continuation byte.
```